src/booking/containers/booking-check/booking-check.component.html

```diff
@@ -7,7 +7,7 @@ <h1 class="govuk-heading-l">
   <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">Location</dt>
     <dd class="govuk-summary-list__value">
-      <p class="govuk-body">{{bookingProcess.locationName}}</p>
+      <p class="govuk-body">{{bookingProcess.location.court_name}}</p>
```

Comment:
Not sure if this is by design, but I'd prefer the camelCase consistency

Reply:
I'd definitely prefer it too, however the location model is using LocationByEPIMSModel in the common lib and this is the structure returned from the API endpoint.